docs/sources/flow/reference/components/loki.source.cloudflare.md

```diff
@@ -39,11 +39,11 @@ Name            | Type                 | Description          | Default | Requir
 `workers`       | `int`                | The number of workers to use for parsing logs.     |  `3` | no
 `pull_range`    | `duration`           | The timeframe to fetch for each pull request.      | `"1m"` | no
 `fields_type`   | `string`             | The set of fields to fetch for log entries.        | `"default"` | no
+`additional_fields` | `list(string)`       | The addional list of fields to supplement those provided via `fields_type`. |  | no
 
 
-By default `loki.source.cloudflare` fetches logs with the `default` set of
-fields. Here are the different sets of `fields_type` available for selection,
-and the fields they include:
+By default `loki.source.cloudflare` fetches logs with the `default` set of fields + the fields passed through `additional_fields`. 
+Here are the different sets of `fields_type` available for selection, and the fields they include:
 
```

Comment:
I know it was there before, but I've noticed that:
- Line 55 doesn't seem to be finished, missing closing " or maybe fields were cut?
- Line 65 has additional `k` at the end

Could you review if these are correct, matching the code?

Reply:
Looks like 2 extra chars on line 65....`` `k``